runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ValidateRunnerXlangTest.java

```diff
@@ -54,15 +54,36 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** Test External transforms. */
+/**
+ * Runner Validation Test Suite for Cross-language Transforms.
+ *
+ * <p>As per Beams's Portability Framework design, Cross-language transforms should work out of the
+ * box. In spite of this, there always exists a possibility of rough edges existing. It could be
+ * caused due to unpolished implementation of any part of the execution code path, for example: –>
+ * Transform expansion [SDK] –> Pipeline construction [SDK] –> Cross-language artifact staging
```

Comment:
Does "->" have a special meaning ? If not we should just use comma here.

Reply:
Oh wow. Technically, "–> Transform expansion [SDK]" is supposed to be one bullet point (and applies to others as well). Seems like spotless didn't like it and put them as you see it. I'll change it to commas.